go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/prometheus/tsdb v0.7.1
 	github.com/rjeczalik/notify v0.9.1
 	github.com/rs/cors v1.7.0
-	github.com/scroll-tech/da-codec v0.1.1-0.20240822151711-9e32313056ac
+	github.com/scroll-tech/da-codec v0.1.1-0.20241004100020-91ac897e28fc
```

Comment:
Why not upgrade da-codec version.

Reply:
will update this dependency with final version once da-code pr is merged.

Reply:
OK. will upgrade it in another PR based on `syncUpstream/active` branch.